zerver/lib/actions.py

```diff
@@ -7649,6 +7649,13 @@ def do_invite_users(
     notify_invites_changed(user_profile.realm)
 
 
+def get_invitation_expiry_date(confirmation_obj: Confirmation) -> Union[int, str]:
+    expiry_date = confirmation_obj.expiry_date
+    if expiry_date is None:
+        return "never"
+    return datetime_to_timestamp(expiry_date)
```

Comment:
Do we want to return the string `never` here?  I would have thought `None/null` would be better in the API for a non-expiring link.

Reply:
Updated it to return `None`.